OpenRA.Game/Primitives/Color.cs

```diff
@@ -83,13 +51,8 @@ public static Color FromAhsv(float h, float s, float v)
 
 		public void ToAhsv(out int a, out float h, out float s, out float v)
 		{
-			var ll = 2 * GetBrightness();
-			var ss = GetSaturation() * ((ll <= 1) ? ll : 2 - ll);
-
 			a = A;
-			h = GetHue() / 360f;
-			s = (2 * ss) / (ll + ss);
-			v = (ll + ss) / 2;
+			(h, s, v) = RgbToHsv(R, G, B);
```

Comment:
Looks like this isn't converting bytes to float first, so I expect the output V at least to be bogus.